packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -270,23 +271,22 @@ export class SliceMachineInitProcess {
 								)}`;
 							},
 						},
-						// TODO(DT-1386): Handle starter detection for Segment
-						// {
-						// 	title: "Detecting starter...",
-						// 	task: async (_, task) => {
-						// 		this.context.starterID = await detectStarterId(
-						// 			this.manager.cwd,
-						// 		);
-
-						// 		if (this.context.starterID) {
-						// 			task.title = `Detected starter ${chalk.cyan(
-						// 				this.context.starterID,
-						// 			)}`;
-						// 		} else {
-						// 			task.title = "No starter detected";
-						// 		}
-						// 	},
-						// },
+						{
+							title: "Detecting starter...",
+							task: async (_, task) => {
+								this.context.starterID = await detectStarterId(
+									this.manager.cwd,
+								);
+
+								if (this.context.starterID) {
+									task.title = `Detected starter ${chalk.cyan(
+										this.context.starterID,
+									)}`;
+								} else {
+									task.title = "No starter detected";
```

Comment:
hmm, they'll no what a starter is so I guess this won't cause any panic

Reply:
Discussed with Sam, it's ok like that 👍

Reply:
It appears for a split second anyway because the task drawer collapse right after

Reply:
Good one, true